tempConversion/tempConversion.js

```diff
@@ -1,13 +1,13 @@
-const ftoc = function(f) {
- return Math.round((f - 32) * (5/9) * 10) / 10;
+const convertToCelsius = function(fahrenheit) {
+   return Math.round((fahrenheit - 32) * (5/9) * 10) / 10;
```

Comment:
Very nitty nitpick, but can we just fix this spacing to match line 6?

Reply:
Fixed with https://github.com/TheOdinProject/javascript-exercises/pull/300/commits/171ac956cfa63804fd8082fffb4db508ced8ea65